ChangeLog

```diff
@@ -10,6 +10,10 @@ Release date: TBA
   Put new features here and also in 'doc/whatsnew/2.14.rst'
 
 
+* Fixed false positive for ``unused-argument`` when a ``nonlocal`` name is used
```

Comment:
Shouldn't this target `2.13.1`? @Pierre-Sassoulas

Reply:
How about we backport all bug fixes and false positives in last minor, keeping only features and false negatives in the next minor/major ?

Reply:
I think you meant "in last patch" (_Sem ver names are awful_), but you I agree!

Reply:
In my head it's always "breaking change", "major", "minor" instead of "major", "minor" "patch" 😄